### PR TITLE
Add ECS walkthrough and update the readme for colorapp

### PR DESCRIPTION
**Pull request by subfuzion**
*Description of changes:*

Provides a comprehensive walkthrough for getting the Color App up and running with App Mesh. The colorapp README is cleaned up a bit and provides a link to the walkthrough.

By submitting this pull request, I confirm that you can use, modify, copy, and redistribute this contribution, under the terms of your choice.

### CODE REVIEW

File: walkthroughs/ecs/regions.md [hunk 1]
```diff
@@ -0,0 +1,27 @@
...
+| Asia Pacific (Sydney)      | ap-southeast-2 |
+| Asia Pacific (Tokyo)       | ap-northeast-1 |
+| Asia Pacific (Seoul)       | ap-northeast-2 |
+| Asia Pacific (Osaka-Local) | ap-northeast-3 |
+| China (Beijing)            | cn-north-1     |
+| China (Ningxia)            | cn-northwest-1 |
+| AWS GovCloud (US-East)     | us-gov-east-1  |
+| AWS GovCloud (US-West)     | us-gov-west-1  |
+
+[AWS regions]: https://github.com/aws/aws-app-mesh-roadmap/issues/1
```
Comment (bcelenza) [inline walkthroughs/ecs/regions.md]: A few things here:

1. App Mesh is not available in all of these regions. See the issue you linked at the bottom for the accurate list (**checked** boxes only).
2. Does it make sense to move this to the main repository readme?

Comment (subfuzion) [reply walkthroughs/ecs/regions.md]: It's missing a 3rd column that had Xs for the actual supported regions, but I'm going to remove this altogether and just point to the official link to check.

Comment (subfuzion) [reply walkthroughs/ecs/regions.md]: Regarding making it the main README, @kiranmeduri also suggested that and I'm fine with that. I'll make the change and push an update shortly.

**Comment by subfuzion** [comment]
@bcelenza Which links looked problematic to you? The relative links should work once merged to `master`, which you can see [here](https://github.com/subfuzion/appmesh-blog/blob/master/colorapp/walkthrough.md) where I worked on the draft.

**Comment by subfuzion** [comment]
@bcelenza I've moved over everything to the `colorapp` directory (and placed images in a `img` subdirectory so the `colorapp` isn't cluttered with a bunch of png files). I've removed the custom supported regions table to link instead to the official docs. Ready for your review now.

**Pull request merged**